desktop/core/src/desktop/api_public_urls.py

```diff
@@ -72,6 +72,14 @@
       api_public.autocomplete,
       name="api_autocomplete_nested",
   ),
+
+  # HS2, RDBMS, JDBC
```

Comment:
Also to add [/sample URLs for SQLite](https://github.com/cloudera/hue/blob/3bf8ce3fa0c3a1987b13e82d565ad8e1d37d9cb8/desktop/libs/notebook/src/notebook/urls.py#L113-L119)? @romainr

Reply:
Can skip for now!